lib/active_shipping/carriers/fedex.rb

```diff
@@ -660,8 +660,15 @@ def parse_tracking_response(response, options)
           country  = address.at('CountryCode').try(:text)
 
           location = Location.new(:city => city, :state => state, :postal_code => zip_code, :country => country)
-          description = event.at('EventDescription').text
-          type_code = event.at('EventType').text
+
+          event_description = event.at('EventDescription').text
+          event_type_code = event.at('EventType').text
+
+          exception_status_code = event.at('StatusExceptionCode').try(:text)
```

Comment:
It's okay for this to be `nil`, and `try!` wouldn't serve any particular purpose since `.at()` will always return either a String or `nil`.

That said, I'm okay with enforcing good practices and using `try!` anyway.